From df20127c9b6596fada764eeca97b482e041127b8 Mon Sep 17 00:00:00 2001
From: Pullakavi Srinivas <spullaka@codeaurora.org>
Date: Tue, 18 Sep 2018 14:05:52 +0530
Subject: [PATCH 1/2] base: Add additional builtins to scan ids.

Use the extended builtin displays range to
query connection status.

CRs-Fixed: 2319037
Change-Id: Icbea9f40bbb135939e25e59e986856cdc67cd7a2
---
 .../java/com/android/server/display/LocalDisplayAdapter.java   | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/display/LocalDisplayAdapter.java b/services/core/java/com/android/server/display/LocalDisplayAdapter.java
index 16d82df4dd5b..b0813fb1a4e3 100644
--- a/services/core/java/com/android/server/display/LocalDisplayAdapter.java
+++ b/services/core/java/com/android/server/display/LocalDisplayAdapter.java
@@ -80,7 +80,8 @@ final class LocalDisplayAdapter extends DisplayAdapter {
 
         mHotplugReceiver = new HotplugDisplayEventReceiver(getHandler().getLooper());
 
-        for (int builtInDisplayId : BUILT_IN_DISPLAY_IDS_TO_SCAN) {
+        for (int builtInDisplayId = SurfaceControl.BUILT_IN_DISPLAY_ID_MAIN;
+             builtInDisplayId <= SurfaceControl.BUILT_IN_DISPLAY_ID_EXT_MAX; builtInDisplayId++) {
             tryConnectDisplayLocked(builtInDisplayId);
         }
     }
-- 
2.17.1

